src/pages/in/like/_id/index.vue

```diff
@@ -151,17 +151,20 @@ export default {
   asyncData(ctx) {
     return Promise.all([
       mixin.asyncData(ctx),
-      () => {
+      new Promise((resolve) => {
```

Comment:
Don't need `new Promise(...)`, just put a `Promise.resolve()` in the `else` case

Reply:
or use mean using `Promise.resolve().then(...)`?